exporter/exporterhelper/common.go

```diff
@@ -86,6 +86,19 @@ func WithRetry(config configretry.BackOffConfig) Option {
 	}
 }
 
+// WithRetry overrides the default configretry.BackOffConfig for an exporter.
+// The default configretry.BackOffConfig is to disable retries.
+func WithConcurrency(config ConcurrencySettings) Option {
+	return func(o *baseExporter) error {
+		if !config.Enabled {
+			o.exportFailureMessage += " Try enabling retry_on_failure config option to retry on retryable errors."
+			return nil
+		}
```

Comment:
I think you can remove this, or you could add "Try enabling the concurrency config option to send more requests in parallel." but generally I dislike these very-wordy messages and think they tend to over-simplify the response a user will take.

Reply:
Thanks for the review! Hmm changed the message, but I think I will leave it in for now to remain consistent with other exporterhelper options

Reply:
Actually removed this error now since I removed config `Enabled` option